{
  "uid" : "d5ee4d587ba07109",
  "name" : "test_revoked_approval",
  "fullName" : "tests.brownie.token.tests.test_transferFrom#test_revoked_approval",
  "historyId" : "8c97c4bdddac6f4b31a5bbd91220f784",
  "time" : {
    "start" : 1638414589477,
    "stop" : 1638414589477,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "ValueError: Gas estimation failed: 'HTTPConnectionPool(host='test3-solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbe4c513340>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.",
  "statusTrace" : "brownie_project = <Project 'TokenProject'>\naccounts = <brownie.network.account.Accounts object at 0x7fe280d0b370>\n\n    @pytest.fixture(scope=\"module\")\n    def token(brownie_project, accounts):\n>       return brownie_project.Token.deploy(\"Test Token\", \"TST\", 18, 1e21, {'from': accounts[0]})\n\ntests/brownie/token/tests/conftest.py:49: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/contract.py:600: in __call__\n    return tx[\"from\"].deploy(\n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/account.py:508: in deploy\n    receipt, exc = self._make_transaction(\n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/account.py:725: in _make_transaction\n    raise VirtualMachineError(e) from None\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = VirtualMachineError(ValueError(\"Gas estimation failed: 'HTTPConnectionPool(host='test3-solana', port=8899): Max retrie... resolution'))'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.\"))\nexc = \"Gas estimation failed: 'HTTPConnectionPool(host='test3-solana', port=8899): Max retries exceeded with url: / (Caused ...me resolution'))'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.\"\n\n    def __init__(self, exc: ValueError) -> None:\n        try:\n            exc = exc.args[0]\n        except Exception:\n            pass\n    \n        if isinstance(exc, dict) and \"message\" in exc:\n            if \"data\" not in exc:\n                raise ValueError(exc[\"message\"]) from None\n    \n            self.message: str = exc[\"message\"].rstrip(\".\")\n    \n            if isinstance(exc[\"data\"], str):\n                # handle parity exceptions - this logic probably is not perfect\n                if \"0x08c379a0\" in exc[\"data\"]:\n                    revert_type, err_msg = [i.strip() for i in exc[\"data\"].split(\"0x08c379a0\", 1)]\n                    err_msg = eth_abi.decode_abi([\"string\"], HexBytes(err_msg))\n                    err_msg = f\"{revert_type} '{err_msg}'\"\n                elif exc[\"data\"].endswith(\"0x\"):\n                    err_msg = exc[\"data\"][:-2].strip()\n                else:\n                    err_msg = exc[\"data\"]\n                raise ValueError(f\"{self.message}: {err_msg}\") from None\n    \n            try:\n                txid, data = next((k, v) for k, v in exc[\"data\"].items() if k.startswith(\"0x\"))\n            except StopIteration:\n                raise ValueError(exc[\"message\"]) from None\n    \n            self.txid: str = txid\n            self.source: str = \"\"\n            self.revert_type: str = data[\"error\"]\n            self.pc: Optional[str] = data.get(\"program_counter\")\n            if self.pc and self.revert_type == \"revert\":\n                self.pc -= 1\n    \n            self.revert_msg: Optional[str] = data.get(\"reason\")\n            self.dev_revert_msg = brownie.project.build._get_dev_revert(self.pc)\n            if self.revert_msg is None and self.revert_type in (\"revert\", \"invalid opcode\"):\n                self.revert_msg = self.dev_revert_msg\n            elif self.revert_msg == \"Failed assertion\":\n                self.revert_msg = self.dev_revert_msg or self.revert_msg\n    \n        else:\n>           raise ValueError(str(exc)) from None\nE           ValueError: Gas estimation failed: 'HTTPConnectionPool(host='test3-solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbe4c513340>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.\n\n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/exceptions.py:121: ValueError",
  "flaky" : false,
  "newFailed" : false,
  "newBroken" : true,
  "newPassed" : false,
  "retriesCount" : 0,
  "retriesStatusChange" : false,
  "beforeStages" : [ {
    "name" : "configure_neon_network",
    "time" : {
      "start" : 1638414565014,
      "stop" : 1638414565015,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "brownie_project",
    "time" : {
      "start" : 1638414566601,
      "stop" : 1638414584930,
      "duration" : 18329
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "accounts",
    "time" : {
      "start" : 1638414585447,
      "stop" : 1638414585723,
      "duration" : 276
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "setup_env",
    "time" : {
      "start" : 1638414585447,
      "stop" : 1638414585447,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "token",
    "time" : {
      "start" : 1638414588098,
      "stop" : 1638414588438,
      "duration" : 340
    },
    "status" : "broken",
    "statusMessage" : "ValueError: Gas estimation failed: 'HTTPConnectionPool(host='test3-solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbe4c513340>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.\n",
    "statusTrace" : "  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/_pytest/fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/neon-compatibility/neon-compatibility/tests/brownie/token/tests/conftest.py\", line 49, in token\n    return brownie_project.Token.deploy(\"Test Token\", \"TST\", 18, 1e21, {'from': accounts[0]})\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/contract.py\", line 600, in __call__\n    return tx[\"from\"].deploy(\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/account.py\", line 508, in deploy\n    receipt, exc = self._make_transaction(\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/account.py\", line 725, in _make_transaction\n    raise VirtualMachineError(e) from None\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/exceptions.py\", line 121, in __init__\n    raise ValueError(str(exc)) from None\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "accounts::0",
    "time" : {
      "start" : 1638414589948,
      "stop" : 1638414589948,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "feature",
    "value" : "Brownie"
  }, {
    "name" : "parentSuite",
    "value" : "tests.brownie.token.tests"
  }, {
    "name" : "suite",
    "value" : "test_transferFrom"
  }, {
    "name" : "host",
    "value" : "fv-az123-322"
  }, {
    "name" : "thread",
    "value" : "3469-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.brownie.token.tests.test_transferFrom"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 12,
        "broken" : 5,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 17
      },
      "items" : [ {
        "uid" : "2a31105bd93ba97",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/147//#testresult/2a31105bd93ba97",
        "status" : "failed",
        "statusDetails" : "AssertionError: Transaction did not revert",
        "time" : {
          "start" : 1638399369642,
          "stop" : 1638399375065,
          "duration" : 5423
        }
      }, {
        "uid" : "c3be23f08202e386",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/2//#testresult/c3be23f08202e386",
        "status" : "failed",
        "statusDetails" : "AssertionError: Transaction did not revert",
        "time" : {
          "start" : 1638376002845,
          "stop" : 1638376007235,
          "duration" : 4390
        }
      }, {
        "uid" : "c01e470d525ab6",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/146//#testresult/c01e470d525ab6",
        "status" : "failed",
        "statusDetails" : "AssertionError: Transaction did not revert",
        "time" : {
          "start" : 1638363660790,
          "stop" : 1638363666192,
          "duration" : 5402
        }
      }, {
        "uid" : "8c4a5fdf107ebddf",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/145//#testresult/8c4a5fdf107ebddf",
        "status" : "failed",
        "statusDetails" : "AssertionError: Transaction did not revert",
        "time" : {
          "start" : 1638328237280,
          "stop" : 1638328241875,
          "duration" : 4595
        }
      }, {
        "uid" : "16c06d55a1ef5f76",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/143//#testresult/16c06d55a1ef5f76",
        "status" : "failed",
        "statusDetails" : "AssertionError: Transaction did not revert",
        "time" : {
          "start" : 1638318704884,
          "stop" : 1638318711045,
          "duration" : 6161
        }
      }, {
        "uid" : "8a5fb51edb9edc5c",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/141//#testresult/8a5fb51edb9edc5c",
        "status" : "failed",
        "statusDetails" : "AssertionError: Transaction did not revert",
        "time" : {
          "start" : 1638285876997,
          "stop" : 1638285883653,
          "duration" : 6656
        }
      }, {
        "uid" : "4203395fd155ae07",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/137//#testresult/4203395fd155ae07",
        "status" : "failed",
        "statusDetails" : "AssertionError: Transaction did not revert",
        "time" : {
          "start" : 1638265813839,
          "stop" : 1638265819371,
          "duration" : 5532
        }
      }, {
        "uid" : "bc78c1ca2efde295",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/140//#testresult/bc78c1ca2efde295",
        "status" : "broken",
        "statusDetails" : "ValueError: Transaction simulation failed: Error processing Instruction 0: insufficient funds for instruction",
        "time" : {
          "start" : 1638269198259,
          "stop" : 1638269198259,
          "duration" : 0
        }
      }, {
        "uid" : "4bfbe6185cb782fb",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/138//#testresult/4bfbe6185cb782fb",
        "status" : "broken",
        "statusDetails" : "ValueError: Gas estimation failed: 'HTTPConnectionPool(host='test3-solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f4107a10c10>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.",
        "time" : {
          "start" : 1638265727316,
          "stop" : 1638265727316,
          "duration" : 0
        }
      }, {
        "uid" : "6b86349d78fcd932",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/136//#testresult/6b86349d78fcd932",
        "status" : "broken",
        "statusDetails" : "ValueError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.",
        "time" : {
          "start" : 1638246927538,
          "stop" : 1638246927538,
          "duration" : 0
        }
      }, {
        "uid" : "769f08aecb9354b5",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/135//#testresult/769f08aecb9354b5",
        "status" : "failed",
        "statusDetails" : "AssertionError: Transaction did not revert",
        "time" : {
          "start" : 1638241848932,
          "stop" : 1638241855171,
          "duration" : 6239
        }
      }, {
        "uid" : "b50a2b37730801d2",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/134//#testresult/b50a2b37730801d2",
        "status" : "failed",
        "statusDetails" : "AssertionError: Transaction did not revert",
        "time" : {
          "start" : 1638232201586,
          "stop" : 1638232207818,
          "duration" : 6232
        }
      }, {
        "uid" : "3079cbc3a9a68cea",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/133//#testresult/3079cbc3a9a68cea",
        "status" : "failed",
        "statusDetails" : "AssertionError: Transaction did not revert",
        "time" : {
          "start" : 1638219689615,
          "stop" : 1638219695061,
          "duration" : 5446
        }
      }, {
        "uid" : "19305f0b969795c7",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/131//#testresult/19305f0b969795c7",
        "status" : "failed",
        "statusDetails" : "AssertionError: Transaction did not revert",
        "time" : {
          "start" : 1638203556674,
          "stop" : 1638203561269,
          "duration" : 4595
        }
      }, {
        "uid" : "90168476d88c8348",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/129//#testresult/90168476d88c8348",
        "status" : "failed",
        "statusDetails" : "AssertionError: Transaction did not revert",
        "time" : {
          "start" : 1638200071076,
          "stop" : 1638200076632,
          "duration" : 5556
        }
      }, {
        "uid" : "4a9714a945e251a0",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/128//#testresult/4a9714a945e251a0",
        "status" : "broken",
        "statusDetails" : "ValueError: Gas estimation failed: 'Transaction simulation failed: Error processing Instruction 1: invalid account data for instruction'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.",
        "time" : {
          "start" : 1638198575303,
          "stop" : 1638198575303,
          "duration" : 0
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "d5ee4d587ba07109.json",
  "parameterValues" : [ ]
}